docker-compose.test.yml

```
@@ -46,10 +48,10 @@
       - '8080:8080'
     restart: unless-stopped
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${MOCKOON_PORT}"]
+      test: [ "CMD", "curl", "-f", "http://${WEBSITE_DOMAIN}:${MOCKOON_PORT}" ]
 
   k6:
-    profiles: ["load"]
+    profiles: [ "load" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)